### PR TITLE
Add JDBC Spring Boot sample

### PR DESCRIPTION
**Pull request by vpavic**
I've signed the CLA.

### CODE REVIEW

File: samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * Initializes the H2 {@link WebServlet} so we can access our in memory database from the
+ * URL "/h2".
+ *
+ * @author Rob Winch
+ */
+@Configuration
+public class H2Initializer {
+
+	@Bean
```
Comment (rwinch) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java]: Is this necessary with Boot? I thought this was automatically setup to `/h2-console` via `H2ConsoleAutoConfiguration`?

File: samples/httpsession-jdbc-boot/src/main/java/sample/config/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * @author Rob Winch
+ * @author Vedran Pavic
+ */
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+	@Autowired
+	public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+		auth.inMemoryAuthentication().withUser("user").password("password").roles("USER");
```
Comment (rwinch) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/SecurityConfig.java]: Can we add no format tags around these methods so they are formatted correctly?

**Comment by rwinch** [comment]
Thanks for the PR @vpavic! You are really rocking Spring Session as of late :) I provided a few comments inline.

File: samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java (hunk 1 above)
Comment (vpavic) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java]: You're absolutely correct. To make my oversight more comical I've just used Boot's H2 console auto-configuration in one of my projects a few days ago :)

Comment (vpavic) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java]: On a sidenote - websocket sample could also be changed to use auto-configuration for H2 console.

Comment (rwinch) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java]: Happens to me sometimes too. It just shows you are a long time Boot user :)

File: samples/httpsession-jdbc-boot/src/main/java/sample/config/SecurityConfig.java (hunk 2 above)
Comment (vpavic) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/SecurityConfig.java]: Yes, we can. I've copied the class from the original boot sample (together with formatting) and added what I needed for this sample.

File: samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java (hunk 1 above)
Comment (rwinch) [inline samples/httpsession-jdbc-boot/src/main/java/sample/config/H2Initializer.java]: > On a sidenote - websocket sample could also be changed to use auto-configuration for H2 console.

Good point! I created #457

**Comment by vpavic** [comment]
Thank you @rwinch :)

**Comment by vpavic** [comment]
@rwinch I've updated the PR.

**Comment by vpavic** [comment]
@rwinch I've updated the PR... again :)
This time to replace Bootstrap CSS files with webjars.

**Comment by rwinch** [comment]
Nice work! Thanks for the PR..This is now merged into master

**Pull request merged**